lib/Screen/HomeScreen.dart

```diff
@@ -0,0 +1,168 @@
+import 'package:flutter/material.dart';
+import 'package:hatspace/Screen/VerificationScreen.dart';
+import 'package:hatspace/main.dart';
+
+class MyHomePage extends StatefulWidget {
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  bool _isButtonDisable = false;
+  String emailValue = "";
+
+  @override
+  Widget build(BuildContext context) {
+    return Theme(
+        data: ThemeData(
+            elevatedButtonTheme: ElevatedButtonThemeData(
+                style: ButtonStyle(
+                  fixedSize: MaterialStateProperty.all<Size>(
+            const Size.fromHeight(45),
+          ),
+          backgroundColor: MaterialStateProperty.all<Color>(
+              const Color.fromRGBO(255, 255, 255, 1)),
+            ))),
+        child: Scaffold(
+          resizeToAvoidBottomInset: false,
+          backgroundColor: Theme.of(context).primaryColor,
+          body: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
+                Container(
+                    // color:Colors.amberAccent,
+                    margin: const EdgeInsets.only(top: 60.0, bottom: 20.0),
+                    child: const Text(
+                      "Space App",
+                      textAlign: TextAlign.center,
+                      style: TextStyle(
+                          fontSize: 16,
+                          fontWeight: FontWeight.w800,
+                          color: Colors.white),
+                    )),
+                Container(
+                    // color: Colors.amberAccent,
+                    // color:Colors.amberAccent,
+                    child: Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    const Text(
+                      "Space",
+                      style: TextStyle(
+                          color: Colors.white,
+                          fontSize: 65,
+                          fontWeight: FontWeight.bold),
```

Comment:
use `Theme.of(context).textTheme.headline2`, either healine2 or 1, need to check the size.